### PR TITLE
Container image tags action

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

Allow to specify a container registry for container-image-tags action.

**Why**:

Without a registry always the default registry from dockerhub is used.

**How**:

**Checklist**:

- [x] Tests
- [ ] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

File: container-image-tags/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+names.
+
+## Inputs
+
+### `strip-tag-prefix`
+The tag prefix to strip i.e v1.2.3 -> 1.2.3 (default 'v') if GitHub reference is
+a tag.
+
+### `image-name`
+The image name to use. By default it is derived from the repository name.
```
Comment (y0urself) [inline container-image-tags/README.md:13]:
```suggestion
The image name to use. By default it is derived from the repository name.
```

e.g.?
`<orga>/<repo>:<branch/tag>`?

File: container-image-tags/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### `registry`
+A container registry to use for naming the container image tags
+
+## Outputs
+
+### `image-tags`
+
+A comma separated string for container image tag names that should be used for
```
Comment (y0urself) [inline container-image-tags/README.md:22]: will they be sorted by the order they are given?

File: container-image-tags/README.md (hunk 1 above)
Comment (bjoernricks) [reply container-image-tags/README.md:13]: I've extended the README to explain how the tags are generated.

File: container-image-tags/README.md (hunk 2 above)
Comment (bjoernricks) [reply container-image-tags/README.md]: No because at the moment only for the stable branch two tags are created.

**Review by y0urself — APPROVED** [review]

**Pull request merged**